docs/docs/references/users/README.md

```diff
@@ -214,3 +214,69 @@ Updating a user's _custom data_ will overwrite (not merge) its original content
 :::
 
 ### Identities
+
+_Identities_ contain the user info retrieved from social sign-in
+(a.k.a. sign-in with a social connector).
+
+The user info from individual social identity providers varies by provider,
+but it will typically include the following:
+
+- Name of this provider, such as lowercase "facebook", "google", or "wechat"
+- User's unique identifier for this provider
+- User's name
+- User's verified email
+- User's avatar
+
+The user may link their account to multiple social identity providers via social sign-in, such as Facebook, Google, and Wechat;
```

Comment:
their?

Reply:
Improve by Grammarly
<img width="925" alt="image" src="https://user-images.githubusercontent.com/10594507/176816695-ebf1de81-eb40-4980-a313-8af266f4d0e4.png">

Reply:
lol, but it's obvious wrong

Reply:
Updated to 

```
The user's account may be linked to multiple social identity providers via social sign-in
```